.github/workflows/inference_cache_stable_diffusion.yml

```diff
@@ -0,0 +1,52 @@
+name: Optimum neuron SD inference cache builder
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Schedule the workflow to run every Saturday at midnight UTC
+    - cron: '0 0 * * 6'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+
+jobs:
+  cache:
+    name: Create optimum-neuron inference cache
+    runs-on:
+      group: aws-inf2-8xlarge
```

Comment:
Is this group using the latest driver corresponding to 2.19.1 ?

Reply:
I think so, we have bump to 2.19.1 for a while(since #661 ), and the CIs for inference have been using the group for inference tests, so shall be ok right?

Reply:
I am not sure they are not up to date, because we did not request the infra team to bump the driver version, so unless they do it automatically, it is still using 2.18 driver.

Reply:
will check with the infra team.